qemu/tests/netperf_stress.py

```diff
@@ -67,6 +68,13 @@ def netperf_test(duration):
     for server in netperf_server:
         s_info = {}
         if server in vms:
+            smp_value = params.get_numeric("smp") or \
```

Comment:
It seems that `smp` and `vcpu_maxcpus` are not set in `netperf_stress_test.cfg`, so I think these two parameters can not get an value.